app/frontend/src/features/messages/Messages.tsx

```diff
@@ -43,28 +43,32 @@ export function HostRequestsSentNotification() {
 }
 
 const labels = {
-  all: "All",
+  //all: "All",
   groupchats: <MessagesNotification />,
   hosting: <HostRequestsReceivedNotification />,
   surfing: <HostRequestsSentNotification />,
-  meet: "Meet",
-  archived: "Archived",
+  //meet: "Meet",
+  //archived: "Archived",
 };
 
 type MessageType = keyof typeof labels;
 
 export default function Messages() {
   const history = useHistory();
-  const { type = "all" } = useParams<{ type: keyof typeof labels }>();
-  const messageType = type in labels ? (type as MessageType) : "all";
+  const { type = "groupchats" } = useParams<{ type: keyof typeof labels }>();
+  const messageType = type in labels ? (type as MessageType) : "groupchats";
 
   const header = (
     <>
       <PageTitle>Messages</PageTitle>
       <TabBar
         value={messageType}
         setValue={(newType) =>
-          history.push(`${messagesRoute}/${newType !== "all" ? newType : ""}`)
+          history.push(
+            `${messagesRoute}/${
+              newType !== "groupchats" ? newType : "groupchats"
```

Comment:
```suggestion
              newType
```

Reply:
What's happening with the other tabs? Are we commenting them out until we actually have content for them or are they going to be removed eventually?

Reply:
Yeah until they have content - at least "All" I'm not sure will be in the beta due to the current separation of host requests and messages.